docs/download-counts.md

```diff
@@ -123,7 +123,7 @@ separated list of packages rather than a single package, e.g.,
 
 `/downloads/point/last-day/npm,express`
 
-__Important:__ As of this writing, 19 April 2017, *scoped* packages are not yet supported in bulk queries. So you cannot request `/downloads/point/last-day/@slack/client,@iterables/map` yet. We *do* plan to support scoped modules in bulk queries as soon as we can make bulk auth checks efficiently. The latter work is in progress.
+__Important:__ *Scoped* packages are not yet supported in bulk queries. So you cannot request `/downloads/point/last-day/@slack/client,@iterables/map` yet. We *do* plan to support scoped modules in bulk queries as soon as we can make bulk auth checks efficiently. The latter work is in progress.
```

Comment:
```suggestion
__Important:__ *Scoped* packages are not yet supported in bulk queries. So you cannot request `/downloads/point/last-day/@slack/client,@iterables/map` yet.
```

Reply:
@MylesBorins It's a little bit sad to see that the plans have changed, but nevertheless that thank you for the merge. 🙏

Reply:
@Simek to be clear the plans haven't necessarily changed. This comment was made in 2017 about 3 years before npm had been acquired by GitHub. I wasn't even aware of this work item prior to reviewing this PR.

Since we haven't even been tracking this internally it seemed inappropriate to leave this comment.

If you want to see us support this a great next step would be to open an discussion on our [feedback repo](https://github.com/npm/feedback)